sdk/include/reactos/mc/bugcodes.mc

```diff
@@ -79,17 +79,15 @@ Severity=Warning
 Facility=System
 SymbolicName=BUGCHECK_MESSAGE_INTRO
 Language=English
-A problem has been detected and ReactOS has been shut down to prevent damage
-to your computer.
+Your PC ran into a problem and needs to restart.
```

Comment:
I'm not sure what's supposed to be better about this string.

Reply:
`ReactOS ran into a problem.`

Reply:
"Huston, we have a Problem !
we need a  boot "
then with a bit charme ;)